fabfile.py

```diff
@@ -379,33 +396,51 @@ def workload(c, upload=None, describe=None):
         if exists == overwrite:
             script = get_value(c, "testy", "unpack_script")
             try: 
-                c.put(upload, "/tmp", preserve_mode=True)
+                c.put(upload, "/tmp/", preserve_mode=True)
             except Exception as e:
                 print(e)
                 print(f"Upload failed for workload '{workload_name}'.")
             else:
-                copy = c.sudo(f"cp /tmp/{upload} {src}", user=user, warn=True)
+                copy = c.sudo(f"cp /tmp/{archived_name} {src}", user=user, warn=True)
                 unpack = c.sudo(f"python3 {script} unpack_archive {src} {dest}", user=user, \
                     warn=True)
                 if copy and unpack: 
                     print(f"Upload succeeded! Workload '{workload_name}' ready for use.")
                 else:
                     print(f"Failed to add '{workload_name}'.")
-                c.sudo(f"rm -f {src} /tmp/{upload}")
+                c.sudo(f"rm -f {src} /tmp/{archived_name}")
 
     # Describes the specified workload by running the describe function as defined in the workload
     # interface file. A workload must be specified for the describe option. 
     if describe:
-        wif = get_value(c, "application", "workload_dir") + f"/{describe}/{describe}.sh"
+        wif = dest + f"/{describe}/{describe}.sh"
         command = wif + " describe"
         result = c.sudo(command, user=user, warn=True)
         if not result: 
             print(f"Unable to describe '{describe}' workload.")
         elif result.stdout == "":
             print(f"No description provided for workload '{describe}'.")
     
+    # Upload a test format config file to the test_format workloads directory ready for use, this 
+    # function takes both relative or absolute paths to your local config file. 
+    if upload_config:
+        config_filename = os.path.basename(upload_config)
+        src = f"{dest}/test_format/{config_filename}"
+
+        try:
+            c.put(upload_config, f"/tmp/", preserve_mode=True)
+        except Exception as e:
+            print(e)
+            print(f"Upload failed for config file '{config_filename}'.")
+        else:
+            copy = c.sudo(f"cp -f /tmp/{config_filename} {src}", user=user, warn=True)
+            if copy:
+                print(f"Upload succeeded for config file '{config_filename}'.")
+            c.sudo(f"rm -f /tmp/{config_filename}")
+
+    
     # If no option has been specified, print the current workload and return as usual.  
-    if not describe and not upload:
+    if not describe and not upload and not upload_config:
```

Comment:
At the end, we do `return current_workload or None`, why don't we simply do:
```
return current_workload
```

Reply:
[ee37c51](https://github.com/wiredtiger/testy/pull/82/commits/ee37c511cd9c92c553a7135fc7ac92e3ee8bab1e)